src/containers/CardDeck.js

```diff
@@ -0,0 +1,337 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import Card from '../components/Card';
+import Container from 'react-bootstrap/Container'
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+
+class CardDeck extends React.Component {
+	
+	state = {
+		content: null,
+		serverRunning: true
+	}
+	
+	componentDidMount(){
+		const frontJSON = [
+			{
+				"doc": {
```

Comment:
hm i guess for the editor to be able to display things, we'll need to wrap things with doc nodes for each block on the frontend after retrieving the blocks for a given view file...